llrt_modules/Cargo.toml

```diff
@@ -23,6 +33,14 @@ rquickjs = { version = "0.6", features = [
 ring = { version = "0.17", optional = true }
 tokio = { version = "1", features = ["rt", "io-util"] }
 
+[target.'cfg(unix)'.dependencies]
+libc = { version = "0.2", optional = true }
+
+[target.'cfg(windows)'.dependencies]
+windows-registry = { version = "0.2", optional = true }
+windows-result = { version = "0.2", optional = true }
+windows-version = { version = "0.1", optional = true }
```

Comment:
Those are official microsoft utils, I started by writing my own low level calls but they are packaging them now so it's better to use that IMO